web/src/app/lists/SimpleListPage.js

```diff
@@ -12,7 +12,8 @@ export default class SimpleListPage extends React.PureComponent {
   }
 
   render() {
-    const { createForm, ...queryProps } = this.props
+    const { createForm, createLabel, ...queryProps } = this.props
```

Comment:
Could you add prop types for these? I think it would look something like (granted I'm not missing something):
```
static propTypes = {
  createForm: p.element,
  createLabel: p.string,
  queryProps: p.object,
}
```